platform/security/secure-boot/src/test/groovy/org/codice/ddf/security/boot/SecureBootSpec.groovy

```diff
@@ -245,68 +245,4 @@ class SecureBootSpec  extends Specification {
         1 * systemService.halt("0")
         exitedDueToSystemServiceFailureToExit == false
     }
-
```

Comment:
❗️ Would annotating these with `@IgnoreIf({ isOsWindows() })` be better? This would actually allow us to verify these conditions on at least some systems like Linux as opposed to loosing these valid test cases completely.

Reply:
I think that's a fantastic idea. Yet again, Spock to the rescue.

Reply:
done